include/coap3/coap_net.h

```diff
@@ -397,6 +397,8 @@ uint16_t coap_new_message_id(coap_session_t *session);
 void coap_free_context(coap_context_t *context);
 
 /**
```

Comment:
It would be good if the 2 deprecated functions were documented in `man/coap_deprecated.txt.in` as well.